lefthook.yml

```
@@ -0,0 +1,51 @@
+# EXAMPLE USAGE:
```

yamllint

[error] wrong new line character: expected \n

(new-lines)